tensorboard/dataclass_compat.py

```diff
@@ -72,11 +76,15 @@ def _migrate_graph_event(old_event, experimental_filter_graph=False):
 
     # TODO(@davidsoergel): Move this stopgap to a more appropriate place.
     if experimental_filter_graph:
-        graph_def = graph_pb2.GraphDef().FromString(graph_bytes)
-        # Use the default filter parameters:
-        # limit_attr_size=1024, large_attrs_key="_too_large_attrs"
-        process_graph.prepare_graph_for_ui(graph_def)
-        graph_bytes = graph_def.SerializeToString()
+        try:
+            graph_def = graph_pb2.GraphDef().FromString(graph_bytes)
+            # Use the default filter parameters:
+            # limit_attr_size=1024, large_attrs_key="_too_large_attrs"
+            process_graph.prepare_graph_for_ui(graph_def)
+            graph_bytes = graph_def.SerializeToString()
+        except message.DecodeError:
+            logger.warning("Could not parse GraphDef.  Skipping.")
```

Comment:
Would it be nicer to print a more information, including how big the GraphDef is, to facilitate user debugging in case necessary?

Reply:
I added the size, but at this point I don't think there's anything else we can say about the graph itself, since it's not parseable.  I guess we have the Event metadata, but I don't think the step number (always 0 for graphs, in practice) is relevant here.